tests/tokenWrapper.test.ts

```diff
@@ -3,6 +3,8 @@ import { artifacts, network, patract } from "redspot";
 import BN from "bn.js";
 import { all } from "@polkadot/api-derive/balances";
 import { hexToU8a } from "@polkadot/util";
+import {ChildProcess, spawn} from 'child_process';
```

Comment:
Please add spaces to imports 
```suggestion
import { ChildProcess, spawn } from 'child_process';
```